python/constellation/satellites/LeCroy/__main__.py

```diff
@@ -0,0 +1,33 @@
+"""
+SPDX-FileCopyrightText: 2024 DESY and the Constellation authors
+SPDX-License-Identifier: EUPL-1.2
+
+Provides the entry point for the LeCroy/LeCrunch satellite
+"""
+
+from typing import Any
+
+from constellation.core.base import EPILOG
+from constellation.core.datasender import DataSenderArgumentParser
+from constellation.core.logging import setup_cli_logging
+
+from .LeCroySatellite import LeCroySatellite
+
+
+def main(args: Any = None) -> None:
```

Comment:
Very minor: if you want to add typing annotations here, it would actually be

```suggestion
def main(args: Sequence[str] | None = None) -> None:
```

(but feel free to ignore - we don't have typing annotations for main anywhere else)